test/unit/dialog-controller.spec.js

```diff
@@ -16,32 +14,38 @@ describe('the Dialog Controller', function () {
     });
   }
 
-  beforeEach(function () {
-    new Container().makeGlobal();
-    settings = { name: 'Test' };
-    dialogController = new DialogController(renderer, settings, Function.prototype, Function.prototype);
+  function createDialogController(settings = {}) {
+    const dialogController = new DialogController(renderer, settings, Function.prototype, Function.prototype);
     dialogController.viewModel = {};
     dialogController.controller = {
       unbind() { }
     };
     dialogController.renderer.hideDialog = Function.prototype;
+    return dialogController;
+  }
+
+  beforeEach(function () {
+    this.dialogController = createDialogController();
```

Comment:
I'm using it like [here](https://jasmine.github.io/2.5/introduction#section-The_%3Ccode%3Ethis%3C/code%3E_keyword).

>  , but naming should take care of this.

What should I change?